Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerCommandCollectionViewCell_Tests.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatUI
+import XCTest
+
+class ChatMessageComposerCommandCellView_Tests: XCTestCase {
+    func test_defaultAppearance() {
+        let view = ChatMessageComposerCommandCellView()
+        view.command = .init(name: "Hello", description: "what", set: "is this", args: "WHAT")
+        AssertSnapshot(view, size: CGSize(width: 300, height: 40), record: true)
+    }
+
+    func test_defaultAppearance_withDirectMessageChannel() {}
+
+    func test_defaultAppearance_withNonDMChannel() {
+        // TODO: https://stream-io.atlassian.net/browse/CIS-652
+    }
+
+    func test_appearanceCustomization_usingUIConfig() {}
+
+    func test_appearanceCustomization_usingAppearanceHook() {}
+
+    func test_appearanceCustomization_usingSubclassing() {}
+}
```

Comment:
This might be nit / unnecessary comment so I am probably sorry :D But this actually isn't part of Removing dead code, it's adding it :trollface:

Reply:
Oh nooo... probably a bad rebase 😅

Reply:
Yeah Forgot to edit this commit 🤦 Fixed it